srcpkgs/exult/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'exult'
+pkgname=exult
+version=1.12.0
+revision=1
+archs="i686* x86_64*"
+build_style=gnu-configure
+configure_args="--prefix=/usr --sysconfdir=/etc --infodir=/usr/share/info --mandir=/usr/share/man --localstatedir=/var --enable-lto --datadir=/usr/share"
```

Comment:
most of the `configure_args` are redundant with `build_style=gnu-configure`.

https://github.com/void-linux/void-packages/blob/31fa4580ed1dfe928c68b6d5a297012da91951cb/common/environment/configure/gnu-configure-args.sh#L11-L12

Reply:
OK I was not sure if the defaults would be overwritten, I'll trim them back to the 2 relevant ones.